ykrt/src/compile/jitc_yk/codegen/mod.rs

```diff
@@ -19,7 +19,7 @@ pub(crate) mod x86_64;
 pub(crate) trait CodeGen<'a> {
     /// Instantiate a code generator for the specified JIT module.
     fn new(
-        jit_mod: &'a jit_ir::Module,
```

Comment:
I **think** this one is OK *here* in the JIT IR, but in other places where you have both kinds of modules around, it could be very confusing.

Hopefully the fact that you are in `jit_ir.rs` should be enough to know that a `m` is a JIT `m` and not an AOT `m`?

Reply:
> Hopefully the fact that you are in jit_ir.rs should be enough to know that a m is a JIT m and not an AOT m?

Exactly my thinking!